src/generated/xproto.rs

```diff
@@ -8590,25 +10154,23 @@ pub const CHANGE_PROPERTY_REQUEST: u8 = 18;
 ///         "XCB Example");
 ///     xcb_flush(conn);
 /// }
+///       
```

Comment:
This looks like another place where the doc should be `.trim()`d

Reply:
Fixed